runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/DataflowRunner.java

```diff
@@ -357,6 +361,36 @@ public static DataflowRunner fromOptions(PipelineOptions options) {
     return new DataflowRunner(dataflowOptions);
   }
 
+  @VisibleForTesting
+  static void validateWorkerSettings(GcpOptions gcpOptions) {
+    Preconditions.checkArgument(
+        gcpOptions.getZone() == null || gcpOptions.getWorkerRegion() == null,
+        "Cannot use option zone with workerRegion.");
```

Comment:
"Cannot use option zone with workerRegion." -> "Cannot use option zone with workerRegion. Prefer workerRegion and workerZone." ?

Reply:
Done.